sql/src/main/java/io/crate/metadata/Schemas.java

```diff
@@ -72,6 +74,7 @@ public Schemas(Settings settings,
         this.docSchemaInfoFactory = docSchemaInfoFactory;
         schemas.putAll(builtInSchemas);
         this.builtInSchemas = builtInSchemas;
+        this.defaultTemplateService = new DefaultTemplateService(clusterService);
```

Comment:
hm not sure if I like this added dependency here (and also triggering the template creation at the cluster listener here). why not making the DefaultTemplateService a real service(lifecycle component) and let it listen itself for state events? (I guess the listener is need at all instead of triggering on startup because if ensures that all is up and running or?)